components/NewRequest/RequestForm.jsx

```diff
@@ -27,6 +27,27 @@ const { Text } = Typography;
 const FORM_NAME = 'ipfs_creation_form_for_mech';
 const FORM_ID = 'myForm';
 
+const ESTIMATED_GAS_LIMIT = 500_000;
+const BUFFER_FACTOR = 1.2;
+
+const getEstimatedGasLimit = async (fn, account) => {
+  if (!account) {
+    throw new Error('Invalid account passed to estimate gas limit');
+  }
+
+  try {
+    const estimatedGas = await fn.estimateGas({ from: account });
```

Comment:
are you doing this because this part here fails?

Reply:
We're doing it because for some reason the gas estimation has started failing – we've seen this in other apps too. So we're adding a fall back.